apps/src/aiComponentLibrary/chatMessage/ChatMessage.tsx

```diff
@@ -89,13 +89,15 @@ const ChatMessage: React.FunctionComponent<ChatMessageProps> = ({
             />
           </div>
         </div>
-        <div
-          className={
-            isTA ? moduleStyles.footerWithOverlay : moduleStyles.footer
-          }
-        >
-          {footer}
-        </div>
+        {footer && (
```

Comment:
Added this to remove an empty div that was adding more space to the bottom of the chat message bubble.